src/lib/openapi/spec/public-signup-token-schema.ts

```diff
@@ -18,28 +18,40 @@ export const publicSignupTokenSchema = {
     ],
     properties: {
         secret: {
+            description: 'The token',
             type: 'string',
+            example: 'a3c84b25409ea8ca1782ef17f94a42fc',
         },
         url: {
             description:
                 'The public signup link for the token. Users who follow this link will be taken to a signup page where they can create an Unleash user.',
             type: 'string',
+            example:
+                'https://sandbox.getunleash.io/enterprise/new-user?invite=a3c84b25409ea8ca1782ef17f94a42fc',
         },
         name: {
+            description: "The token's name",
             type: 'string',
+            example: 'default',
         },
         enabled: {
+            description: 'Whether the token is active',
             type: 'boolean',
+            example: true,
         },
         expiresAt: {
             type: 'string',
             format: 'date-time',
+            example: '2023-04-12T11:13:31.960Z',
         },
         createdAt: {
             type: 'string',
             format: 'date-time',
+            example: '2023-04-12T11:13:31.960Z',
         },
         createdBy: {
+            description: 'The user that created the token',
```

Comment:
It's a bit unclear to me what part of the user this is. Is it their username? Their email? Their displayname? Because it's just a string, I assume it's one of them. Do we use a fallback method if the first isn't available? For instance:

If it's the user's email, but the user doesn't have an email, does it fall back to using the username?